backend/app/Http/Controllers/StudentController.php

```diff
@@ -2,13 +2,36 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Following;
 use App\Models\Student;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Hash;
 
 class StudentController extends Controller
 {
+  public function show($id)
+  {
+    $followingsTable = Following::where('student_id', $id)->where('type', 'follow')->get();
+    $followings = [];
+    foreach ($followingsTable as $following) {
+      array_push($followings, Student::find($following->student_follow));
+    }
+
+    $followersTable = Following::where('student_id', $id)->where('type', 'follower')->get();
+    $followers = [];
+    foreach ($followersTable as $following) {
+      array_push($followers, Student::find($following->student_follower));
+    }
+
+    $response = [
+      'followings' => $followings,
+      'followers' => $followers
+    ];
+
```

Comment:
please separate the following function from profile, do not make it all at once because it causes confusion in PR

Reply:
I'm sorry Sir, you mean like a separate controller for followings (FollowingController) ?